pynumenc_meta.py

```diff
@@ -1,13 +1,10 @@
 """Define meta information about numenc package."""
 
 __title__ = 'numenc'
-__description__ = 'Translate integer and floating-point numbers to/from ' \
-                  'sortable bytes'
+__description__ = 'Convert numbers to/from sortable bytes'
 __url__ = 'https://github.com/Parquery/pynumenc'
 __version__ = '1.0.0'
-__author__ = 'Teodoro Filippini'
-__author_email__ = 'teodoro.filippini@parquery.com'
+__author__ = 'Teodoro Filippini, Marko Ristin'
```

Comment:
I think the convention is to put " and " instead of ", " -- I remember that pypi complained, but please double-check.

Reply:
@mristin the "author" field is free - I'll put in our names joined by "and".

On the iother hand, e-mails need to be comma-separated - see [here](https://packaging.python.org/specifications/core-metadata/#author-email).